src/index.js

```diff
@@ -0,0 +1,25 @@
+const { exec } = require('child_process')
+
+module.exports = function (pluginConfig, config, cb) {
+  exec('git tag', (err, stdout, stderr) => {
+    if (err) return cb(err)
+
+    const versionRegex = /^v((?:[0-9]+\.){2}[0-9]+)$/
+
+    const tags = stdout
+                  .trim()
+                  .split('\n')
+                  .map((t) => t.trim())
+                  .filter(t => versionRegex.test(t))
```

Comment:
I'm not 100% sure if the order of the `git tag` output is reliable, but we could add sorting based on `semver.gt` to make sure.